modules/slc/Animal/t/Cow.t

```diff
@@ -0,0 +1,14 @@
+use strict;
+use warnings;
+
+use Test::More tests => 3;
+
+use_ok( 'Cow' ) || print "Bail out!\n";
+
+is( Cow->sound(), 'moooo', "Cow sounds like moooo" );
+
+my $output = '';
+open TOOUTPUT, '>', \$output or die "Can't open TOOUTPUT: $!";
+select TOOUTPUT;
+Cow->speak();
+is( $output, "a Cow goes moooo!\n", "A cow speaks moo" );
```

Comment:
Nice work. You can also use the [<tt>Test::Output</tt>](http://search.cpan.org/~bdfoy/Test-Output-1.03/lib/Test/Output.pm) module.